### DISCUSSION TITLE
Readme change - unsupported

### DISCUSSION THREAD

**Discussion by stoeps13** [HCL-TECH-SOFTWARE/connections-automation#316] [General]
Hi,
The changes in the README are quite annoying and I wondered about it.

The project was introduced in meetings by OM and lead architects as the recommended way deploying Connections, because it already sets all best practise and tuning parameters. So it was promoted different than described now.

Now people started creating PR to help the project to fulfill this and you move it to an unsupported state. From recommended to only for POC.

Sad that the new open projects are made for the community and then PRs are not merged for weeks, because this makes work necessary on internal repos. To be honest having different repos internal and external can't work and produce a large overhead.

I'm really sorry if anybody used this project after I told him/her that I use it. My appologies for this.

Christoph

**Comment by pln-git4011** [comment]
Our intent is not to change the level of support but only to inform our customers of the changes to how the repro is monitored. There will be updates for each CNX 8.0 CR as usual, the only change is that the discussion board will not be monitored as closely as in the past. There is a backlog of PRs that needs to be reviewed and tested, we are working on improving the process to reduce the backlog. Not every PR raised fits the requirement that all changes need to be generic not tailored to a specific deployment type.

**Comment by stoeps13** [comment]
The REAME says unsupported after itnwas promoted as the recommended way deploying Connections. 
Your explanation on responses does not fit to the changed README.
